spec/controllers/cafe_works_controller_spec.rb

```diff
@@ -84,23 +84,23 @@
       context 'with no user' do
         it 'update worker' do
           patch :update_worker,
-                {
-                    id: cwork_access.to_param,
-                    cafe_work: attributes_for(:assignee, :test, access_code: cwork_access.access_code)
-                }
-          cwork_access.reload
+            {
+            id: cwork_access.to_param,
+            cafe_work: attributes_for(:assignee, :test, access_code: cwork_access.access_code)
+          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.